unit8/deepMenu/scss/styleMenu.scss

```
@@ -0,0 +1,251 @@
+$gray-color: #a5a5a5;
+$orange-color: #d8823b;
+$height-menu-item: 4.2rem;
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: inherit;
+}
+html,
+body {
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    font-size: 62.5%;
+    box-sizing: border-box;
+}
+.d-flex {
+    display: flex;
+}
+.algin-center {
+    align-items: center;
+}
+.p-relative {
+    position: relative;
+}
+.p-absolute {
+    position: absolute;
+}
+.fontSize-12 {
+    font-size: 1.2rem;
+}
+.fontSize-10 {
+    font-size: 1rem;
+}
+.mr-icon {
+    margin-right: 0.8rem;
+}
+.h-icon {
+    width: 1.6rem;
+    text-align: center;
+}
+.container {
+    position: relative;
+    width: 100%;
+    height: 100vh;
+    background-color: #eeeeee;
+}
+.wrapper {
+    position: absolute;
+    width: 12rem;
+    background-color: #ffffff;
+    top: 10px;
+    left: 10px;
+}
+Child-menu {
```

```suggestion
child-menu {
```
- Không được phép viết hoa, từ buổi 1 đã nói rất rõ ràng trong HTML mọi thứ phải viết thường
- `Child-menu` là cái gì: `tag` hay `class`, ... ?